sentences/fr/media_player_HassMediaNext.yaml

```diff
@@ -0,0 +1,8 @@
+language: fr
+intents:
+  HassMediaNext:
+    data:
+      - sentences:
+          - "[morceau|chanson|élément] suivant[e] [sur|pour] [<le>]{name}"
```

Comment:
Make it an expansion rule called `<track>` 
Media player can play more than song, add a few element in the expansion rules such as "film", "épisode", "podcast", "média" (and others if you can think or other type of element media players usually play)